src/editorwidget/widget.ts

```diff
@@ -22,6 +32,21 @@ import {
   ABCWidgetFactory, IDocumentModel, IWidgetFactory, IDocumentContext
 } from '../docregistry';
 
+import 'codemirror/mode/meta';
```

Comment:
These two imports would go before the phosphor ones, and you don't need the `.js` extension.